rma/wizard/rma_delivery.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+import odoo.addons.decimal_precision as dp
+
+
+class RmaReDeliveryWizard(models.TransientModel):
+    _name = 'rma.delivery.wizard'
+    _description = 'RMA Delivery Wizard'
+
+    rma_count = fields.Integer()
+    type = fields.Selection(
+        selection=[
+            ('replace', 'Replace'),
+            ('return', 'Return to customer'),
+        ],
+        string="Type",
+        required=True,
+        default="refund",
+    )
+    product_id = fields.Many2one(
+        comodel_name="product.product",
+        string="Replace Product",
+    )
+    product_uom_qty = fields.Float(
+        string='Product qty',
+        digits=dp.get_precision('Product Unit of Measure'),
+    )
+    product_uom = fields.Many2one(
+        comodel_name="uom.uom",
+        string="Unit of measure",
+    )
+    scheduled_date = fields.Datetime(
+        required=True,
+        default=fields.Datetime.now(),
+    )
+    warehouse_id = fields.Many2one(
```

Comment:
Maybe we should choose an operation type instead, so we ensure warehouse and location at once

Reply:
Thanks!